pkg/combustion/network_test.go

```diff
@@ -0,0 +1,30 @@
+package combustion
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+)
+
```

Comment:
It's unfortunate there isn't a great model to test the Command. I worked around it in the xorriso stuff by returning the command separately from running it. I don't love that either, but IMO it was important to ensure the correct arguments were passed to the command.

Not saying you need to adopt that approach, but is there any reasonably clean way to be able to test the parameters to the generate command?

Reply:
Testing the args themselves is fine the way you did it. Mocking `exec.Command` is another way but it results in higher complexity in tests.